tests/vp_mesh_test.py

```diff
@@ -106,6 +106,8 @@ def test_create_vp_model(tmp_path: Path):
     ):
         VPModel.create(ws, layers=layers, prisms=prisms, unit_property_id="abc")
 
+    assert isinstance(drape.visual_parameters.filter_basement, float)
```

Comment:
Shouldn't you test the default value? 
filter.basement == ...

What happens in the case of a VP mesh containing a visual parameter with no filter_basement value? 
Shouldn't we assign it automatically in @property? with a warning for example?

Reply:
GA always creates it, and our `create` method also always adds it too. So it's expected that Filterbasement is always present/